tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -283,6 +283,10 @@ void test_area(const Solution& solution, const Strahlkorper<Fr>& strahlkorper,
 
   const double irreducible_mass = StrahlkorperGr::irreducible_mass(area);
   CHECK(irreducible_mass == approx(expected_irreducible_mass));
+
+  const double christodoulou_mass = StrahlkorperGr::christodoulou_mass(
+      dimensionless_spin_magnitude, irreducible_mass);
```

Comment:
pass dimensionless_spin_magnitude to test_area(), or pass spin and take magnitude here